components/Tabs/src/TabBarView/MDCTabBarView.h

```diff
@@ -27,6 +27,9 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The currently-selected item in the Tab bar. */
 @property(nullable, nonatomic, strong) UITabBarItem *selectedItem;
 
+/** The content edge insets of the UITabBar. */
+@property(nonatomic) UIEdgeInsets edgeInsets;
```

Comment:
Please name `contentEdgeInsets` and have them flip for RTL.  It will prevent clients from having to do the flip each time.

Reply:
Done.